src/Pages/Cart/index.js

```diff
@@ -1,3 +1,30 @@
+import { useCart } from "../Products/Context/CartContext";
+
+import CartCounter from "./Components/CartCounter";
+import ProductList from "./Components/ProductList";
+import CartTotal from "./Components/CartTotal";
```

Comment:
can make one ```index.js``` & import from there

Reply:
Done as you suggest. thanks you pointing out this